pkg/crd/kubermatic/v1/mla_admin_settings.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//+genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// MLAAdminSetting is the object representing cluster-specific administrator settings
+// for KKP user cluster MLA (monitoring, logging & alerting) stack.
+type MLAAdminSetting struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec MLAAdminSettingSpec `json:"spec"`
+}
+
+// MLAAdminSettingSpec specifies the cluster-specific administrator settings
+// for KKP user cluster MLA (monitoring, logging & alerting) stack.
+type MLAAdminSettingSpec struct {
+	// Cluster is the reference to the user cluster whose MLA settings are defined in this object.
+	Cluster corev1.ObjectReference `json:"cluster"`
```

Comment:
Just realized we don't need this Cluster reference, as this is a namespaces CRD and the namespace defines the user cluster. I'm going to delete it (FYI @aborilov)

Reply:
I think this depends on if your controller logic needs to interact with `Cluster` objects, if yes, then better have a Cluster reference, because deriving cluster name from namespace name may break if we use a different naming pattern for namespaces in the future.

Reply:
OK will keep it for now, I don't think I need this reference for my part but @aborilov let me know whether you need it. Also not sure whether we need full object reference or just a cluster name.

Reply:
Yeah, you are right, we can just use the `Name` in the ObjectReference, and let other fields be empty, and we assgin them values if we need in the future

Reply:
I always can find Cluster from a namespace or using just `Name`. I think `ObjectReference` is useful when we need `MLAAdminSetting` to be enquired when `Cluster` object changed, but I don't think we need that in our case

Reply:
OK, let's keep it